tests/test_views.py

```diff
@@ -109,6 +109,7 @@ def test_bmv_create_form(bmv_instance, url, backlink, app):
 #
 
 xpath_alert = lambda kind, message: f'//div[@class="alert alert-{kind} alert-dismissable"][contains(., "{message}")]'
+xpath_cls = lambda s: ''.join(f'[contains(concat(" ", @class, " "), " {s} ")]' for s in s.split(' ') if s)
```

Comment:
`@class=` literally just checks for string equality; this matches separate class-names instead (like a selector query would)